requirements.in

```diff
@@ -4,3 +4,4 @@ django-redis-sessions==0.6.2
 django-rq==2.4.0
 pip-tools
 django-environ
+magic==0.4.*
```

Comment:
pip-compile has not been run, so this isn't in requirements.txt or requirements_dev.txt